scripts/packaging/createupmpackages.ps1

```diff
@@ -126,7 +126,7 @@ foreach ($entry in $packages.GetEnumerator()) {
         Rename-Item -Path $npmrcFile  -NewName $npmrcBackup
     }
 
-     if (-not ($isLocalBuild)) {
+    if (-not ($isLocalBuild)) {

```

Comment:
Same general comment around Get-Location in this script as well (see:

https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8165/files#r454464718)

Reply:
added parameter to allow for specifying, if not present, it will be determined in the previous manner